packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,349 @@
+import Button from '@celo/react-components/components/Button'
+import Card from '@celo/react-components/components/Card'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Shadow } from '@celo/react-components/styles/styles'
+import variables from '@celo/react-components/styles/variables'
+import React, { useState } from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import {
+  ActivityIndicator,
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { dappsListApiUrlSelector } from 'src/app/selectors'
+import BackButton from 'src/components/BackButton'
+import BottomSheet from 'src/components/BottomSheet'
+import Dialog from 'src/components/Dialog'
+import CustomHeader from 'src/components/header/CustomHeader'
+import DappsExplorerLogo from 'src/icons/DappsExplorerLogo'
+import Help from 'src/icons/navigator/Help'
+import QuitIcon from 'src/icons/QuitIcon'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import useSelector from 'src/redux/useSelector'
+import { navigateToURI } from 'src/utils/linking'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'DAppExplorerScreen'
+
+interface CategoryWithItems {
+  id: string
+  name: string
+  fontColor: string
+  backgroundColor: string
+  items: DappItem[]
+}
+
+interface DappItem {
+  id: string
+  iconUrl: string
+  name: string
+  description: string
+  dappUrl: string
+}
+
+export function DAppsExplorerScreen() {
+  const { t } = useTranslation()
+  const dappsListUrl = useSelector(dappsListApiUrlSelector)
+  const [categoriesWithItems, setcategoriesWithItems] = useState<CategoryWithItems[]>([])
+  const [isHelpDialogVisible, setHelpDialogVisible] = useState(false)
+  const [isBottomSheetVisible, setBottomSheetVisible] = useState(false)
+  const [dappSelected, setDappSelected] = useState<DappItem>()
+
+  if (!dappsListUrl) {
+    return (
+      <View style={styles.centerContainer} testID="DAppExplorerScreen/error">
+        <Text style={fontStyles.large}> Configuration error. </Text>
+        <Text style={fontStyles.large}> dappsListUrl is not set. </Text>
+      </View>
+    )
+  }
+
+  const { loading } = useAsync(
```

Comment:
One thing that's potentially missing is an error. If this fails there will be no items and we won't show the loading indicator anymore but no content either. We can ask Nitya but maybe a simple 'There was an error loading the dApps' could be enough? I'd check with her in any case.